### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ronaldschlenker**
A brief section at the beginning for clarifiying who fsprojects is and what it means to transfer a project from a more legal point of view.

Feel free to change this or discard it completely.

I'm not even sure if it's a good thing emphasising "trust issues" and similar concerns so prominent at the beginning of the document. I find the part above the added lines is quite positive in a way that it adresses the curiosity and community-sense. The added part in contrast addresses potential fears or concerns, and might "destroy" the positive vibe in the introduction section. Maybe it would be enough saying something like: "No ownership or copyright is transferred, and it is maintained by Sergey Tihon and Don Syme", and maybe the fact that it is not a FSSF thing.

Thank you very much @sergey-tihon and @dsyme for all your effort.

### CODE REVIEW

**Comment by ronaldschlenker** [comment]
...or just use a slightly modified version of the words from [this comment](https://github.com/ronaldschlenker/FsHttp/pull/79#issuecomment-965371689) would also do it, and it sounds friendly.

File: README.md [hunk 1]
```diff
@@ -8,6 +8,13 @@ This is an incubation space, and so is slightly anarchic: the idea is to "let ma
 
 Our expectation is that you are seeking collaboration and co-operation, and that you wish to facilitate other people to contribute to your project.
 
+To get a comprehensive understanding, please read [the policies section](#policies) below. Since sharing and collaboration are based on trust and benevolence, here are some important key points of the impact having a project in this space:
+
+* _fsprojects_ is not a legal organization, and it is not part of the [F# Software Foundation](https://fsharp.org), although well-known FSSF members are part of it by hosting or contribution.
+* _fsprojects_ is currently maintained by Sergey Tihon and Don Syme.
```
Comment (sergey-tihon) [inline README.md:14]: We already have it at the bottom 
https://github.com/fsprojects/fsprojects-admin/blob/master/README.md?plain=1#L63

File: README.md [hunk 2]
```diff
@@ -8,6 +8,13 @@ This is an incubation space, and so is slightly anarchic: the idea is to "let ma
...
+* "Transfering" your project to _fsprojects_ does **not** mean that you give away ownership or any rights apart from the ones you grant in your project's license.
```
Comment (sergey-tihon) [inline README.md:15]: I believe that it is also covered by `copyright`
https://github.com/fsprojects/fsprojects-admin/blob/master/README.md?plain=1#L47

File: README.md [hunk 3]
```diff
@@ -8,6 +8,13 @@ This is an incubation space, and so is slightly anarchic: the idea is to "let ma
...
+* Moving your project out of _fsprojects_ is always possible.
```
Comment (sergey-tihon) [inline README.md:16]: This one as well https://github.com/fsprojects/fsprojects-admin/blob/master/README.md?plain=1#L39

**Comment by sergey-tihon** [comment]
@ronaldschlenker can you please elaborate about `trust issues`? was it a real case or a hypothetical?

**Comment by ronaldschlenker** [comment]
It's true that no new content is added in this PR and that the already listed policies cover this PR entirely from a content point of view. The intention of this PR is to summarize some of the policies that potential collaborators might find important and emphasise them by bringing them to the top. Of course this is my personal choice, although I guess that - for example - the question who is behind fsprojects is an important one for many collaborators - but this is currently the last point on the list. Perhaps this applies to the other summary points as well.

**Comment by dsyme** [comment]
@ronaldschlenker I'll close this, and move the mention of @sergey-tihon and myself up the list

**Pull request closed without merging**